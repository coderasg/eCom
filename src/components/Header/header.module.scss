@use './../../variables';

.header {
  padding: 10px;
  border: 1px solid variables.$iconBackground;
  background-color: variables.$white;
  position: sticky;
  top: 0px;
  width: 100%;
  box-sizing: border-box;
}