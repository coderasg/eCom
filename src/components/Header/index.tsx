import React, { ReactNode } from 'react';

import style from './header.module.scss';

type Props = {
  children?: ReactNode;
  className?: string;
};

function Header(props: Props) {
  const { className, children } = props;

  return  (
    <div data-testid="Header-wrapper" className={`${style.header} ${className}`}>
      {children}
    </div>
  );
}

export default Header;
