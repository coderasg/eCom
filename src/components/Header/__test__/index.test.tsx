import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../index';

describe('(Component) Header', () => {
  const defaultProps = {};
  
  test('should pass snapshot test', () => {
    const wrapper = renderer.create(<Header {...defaultProps} />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
