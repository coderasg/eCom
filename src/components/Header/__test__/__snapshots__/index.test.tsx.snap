// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Header should pass snapshot test 1`] = `
<div
  className="undefined undefined"
  data-testid="Header-wrapper"
/>
`;
